js/reducers/report-reducer.ts

```diff
@@ -131,6 +131,7 @@ export default function report(state = new ReportState({}), action?: any) {
       .set("type", "student")
       .set("nowShowing", "student")
       .set("selectedStudentIds", Immutable.fromJS([action.runKey]))
+      .set("hideControls", true)
```

Comment:
I see now that hideControls is basically the same as `type: student`. Whenever we set the type to student we also have to set hideControls to true. It might be better to drop the hideControls property to simplify things, but what you did here is fine too.

Reply:
The `hideControls` is also used int he SelectionCheckbox component so I think it should be left as is otherwise that would need to be refactored.